src/modules/metrics/actions.js

```diff
@@ -22,11 +21,23 @@ export function addMetric(item) {
   }
 }
 
-//spaceId must be done before the metric is removed here.
 export function removeMetrics(ids) {
```

Comment:
any way this could happen on the server?  I realize that would be tricky but it does seem more right to me.